anubis-management-api/anubis/policies/routers.py

```diff
@@ -508,6 +509,24 @@ def create_policy(
                 urllib.parse.quote(
                     policy.access_to)))
         logging.error(e)
+    if os.environ.get('OPA_ENDPOINT'):
+        opa_url = os.environ.get('OPA_ENDPOINT', 'http://127.0.0.1:8181')
+        db_policies = operations.get_policies_by_service_path(
+            db,
+            tenant=fiware_service,
+            service_path_id=db_service_path_id)
```

Comment:
I'm assuming this will work even if there's no service path? think Teadal...

Reply:
also, it looks like `get_policies_by_service_path` comes w/ a default limit param set to 100? what happens if there's more than 100 policies under that tenant/path? again, think Teadal...

Reply:
for now put `-1` as limit